### ISSUE TITLE
nux_desktop_rpm install failing

### ISSUE THREAD

**Issue by corsarof** [Alfresco/chef-alfresco#8]
```
2015/04/20 11:57:14 ui:     amazon: Running handlers:
2015/04/20 11:57:14 ui:     amazon: [2015-04-20T10:57:14+00:00] ERROR: Running exception handlers
2015/04/20 11:57:14 ui:     amazon: Running handlers complete
2015/04/20 11:57:14 ui:     amazon: [2015-04-20T10:57:14+00:00] ERROR: Exception handlers complete
2015/04/20 11:57:14 ui:     amazon: [2015-04-20T10:57:14+00:00] FATAL: Stacktrace dumped to /var/chef/cache/chef-stacktrace.out
2015/04/20 11:57:14 ui:     amazon: Chef Client failed. 166 resources updated in 545.27677711 seconds
2015/04/20 11:57:15 ui:     amazon: [2015-04-20T10:57:14+00:00] ERROR: rpm_package[nux_desktop_rpm] (alfresco::3rdparty line 15) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
2015/04/20 11:57:15 ui:     amazon: ---- Begin output of rpm  -i /var/chef/cache/nux-dextop-release.rpm ----
2015/04/20 11:57:15 ui:     amazon: STDOUT:
2015/04/20 11:57:15 ui:     amazon: STDERR: warning: /var/chef/cache/nux-dextop-release.rpm: Header V4 RSA/SHA1 Signature, key ID 85c6cd8a: NOKEY
2015/04/20 11:57:15 ui:     amazon: error: Failed dependencies:
2015/04/20 11:57:15 ui:     amazon: epel-release is needed by nux-dextop-release-0-5.el7.nux.noarch
2015/04/20 11:57:15 ui:     amazon: ---- End output of rpm  -i /var/chef/cache/nux-dextop-release.rpm ----
2015/04/20 11:57:15 ui:     amazon: Ran rpm  -i /var/chef/cache/nux-dextop-release.rpm returned 1
2015/04/20 11:57:15 ui:     amazon: [2015-04-20T10:57:15+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by maoo** [comment]
Thanks @corsarof , this push https://github.com/maoo/chef-alfresco/commit/af5f877eee6b6da7d28974bd0b04c5977e302360 should fix it. Please close if this works for you.

Make sure you `rm Berksfile.lock` before trying again.
Thanks!
  mao

**Comment by maoo** [comment]
this is fixed on 0.5.1
